juniper_official/Kernel/check-iri-conn-keepalive-dropped.rule

```diff
@@ -0,0 +1,46 @@
+/*
+ * This rule detects any intra-chassis connection (part of Internal Routing
+ * Instance) drops because of keep alive expiry.
+ */
+iceberg {
+    topic kernel.tcpip {
+        rule check-iri-conn-keepalive-dropped {
+            synopsis "Detect IRI connection drops because of Keepalive";
+            description "This rule detects any intra-chassis connection drops because of keep alive expiry";
+            sensor iri-conn-dropped-sensor {
+                iAgent {
+                    file tcpconn.yml;
+                    table tcpconnTable;
+                    frequency 20s;
+                }
+            }
+            trigger report-iri-conn-keepalive-dropped {
```

Comment:
1) Provide trigger description and synopsis detailing sensor info.
2) Trigger frequency is must as default frequency is 10s and trigger gets evaluated once in 10s. but here sensor frequecy is 10 mins hence trigger frequency can be greaterthan equal to 10 mins.
e.g. synopsis "Routing engine GRES kpi";
       description "sets health based on Routing engine CPU utilization for 15 minutes";

Reply:
Addressed